### PR TITLE
static messages

### PR DESCRIPTION
**Pull request by the456gamer**
the messages are currently hardcoded in.
i could move them to a config or messages file, but i think thats future me's problem as part of #13 #12 
on join event for first join

update to 1.19.4, even though it was running without that change :)

uses minimessage

not ready for merging as slack command doesnt give joining info, but can mostly be reviewed

### CODE REVIEW

**Review by polypixeldev — CHANGES_REQUESTED** [review]
Everything else looks good to me

File: src/main/java/com/hackclub/hccore/playerMessages/UnlinkedSlackMessage.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import static net.kyori.adventure.text.minimessage.MiniMessage.miniMessage;
+
+import net.kyori.adventure.text.Component;
+
+public class UnlinkedSlackMessage {
+
+  final static String minimsgSource = """
+      <b><red>Link your MC account with Hack Club Slack</red></b>
+      <gold><b>1.</b></gold> Not on the Slack? Don't worry!
+         Join at <click:open_url:'https://hackclub.com/slack'><hover:show_text:'Open in Browser'>hackclub.com/slack</hover></click>
```
Comment (polypixeldev) [inline src/main/java/com/hackclub/hccore/playerMessages/UnlinkedSlackMessage.java:12]: I think that the slack link should be underlined to signify that it is a clickable link

**Review by polypixeldev — CHANGES_REQUESTED** [review]
The usage info in the plugin.yml should be updated for the new Slack info subcommand.
Everything else looks good rn

File: src/main/java/com/hackclub/hccore/playerMessages/UnlinkedSlackMessage.java (hunk 1 above)
Comment (the456gamer) [reply src/main/java/com/hackclub/hccore/playerMessages/UnlinkedSlackMessage.java:12]: disagree: minecraft underline formating eats into the next line ingame, and there is a hover message anyway

**Review by polypixeldev — APPROVED** [review]

**Pull request merged**